templates/swiftui/BPKIcons.njk

```diff
@@ -24,12 +24,12 @@ public extension BPKIcon {
 }
 
 public extension BPKIcon {
-    static func named(_ iconName: String, bundle: Bundle) -> BPKIcon {
```

Comment:
this method was itnroduced as we before thought this was the problem. the actual api we want is the one from this PR here, one method with a default value.